### PR TITLE
AP_HAL_ChibiOS/hwdef: PodmanH7 FC board target

### PR DESCRIPTION
**Pull request by burgeruser**
Burgeruser will release a new flight controller (PodmanH7).
we added a board definition file and etc.

Thank you.

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/PodmanH7/i2c_gpio.inc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+PB9 I2C1_SDA I2C1
+PB10 I2C2_SCL I2C2
+PB11 I2C2_SDA I2C2
+
+PE10 LED_RED OUTPUT HIGH OPENDRAIN GPIO(90)
+PE12 LED_GREEN OUTPUT HIGH OPENDRAIN GPIO(91)
+PE15 LED_BLUE OUTPUT HIGH OPENDRAIN GPIO(92)
+PC3  LED_AMBER OUTPUT HIGH OPENDRAIN GPIO(94)
+
+PE14 TIM1_CH4 TIM1 PWM(1) GPIO(50)
```
Comment (andyp1per) [inline libraries/AP_HAL_ChibiOS/hwdef/PodmanH7/i2c_gpio.inc:12]: You have an H7 you will want to make some of these bi-directional dshot capable

**Review by andyp1per — CHANGES_REQUESTED** [review]
Add bi-dir support

**Comment by burgeruser** [comment]
Thank you. 
It's interesting! Just now, I was indeed testing a bi-directional ESC (Electronic Speed Controller), and after thorough testing, I confirmed that it is indeed viable. The ESC can successfully communicate via the D-Shot protocol in both directions, enabling us to retrieve the motor's RPM (Revolutions Per Minute) information.

**Comment by peterbarker** [comment]
From a glance I'd suggest JST-GH and CAN, at least.

**Comment by burgeruser** [comment]
> From a glance I'd suggest JST-GH and CAN, at least.

JST-GH is a highly professional interface, equipped with self-locking functionality. Initially, my design utilized the JST-GH1.25 connector, along with a CAN interface. However, for the sake of ease of use for beginners, many of whom are familiar with the Pixhawk design, I aimed to make it more user-friendly. My intention with this version of the flight controller is to facilitate a smoother introduction to Ardupilot for newcomers. Currently, I am working on an extensive Wiki tutorial to encourage more people to join the Ardupilot community, expanding its pool of contributors.

**Pull request closed without merging**